### ISSUE TITLE
Literal logout

### ISSUE THREAD

**Issue by kasp1** [apiko-rest-api/apiko#74]
Is it necessary to have an endpoint that would literally log out the currently logged in user?

Please share your opinions.

**Comment by ilabacheuski** [comment]
I think must be

**Comment by alewiahmed** [comment]
yep, there should be a way to logout the user. That's necessary.
